### PR TITLE
New Package: spampd-2.42 + dependency

### PR DESCRIPTION
**Pull request by crater2150**

### CODE REVIEW

File: srcpkgs/spampd/files/spampd/run [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+[ -r conf ] && . ./conf
+
+exec /usr/bin/spampd --port=${PORT:=10025} \
```
Comment (xtraeme) [inline srcpkgs/spampd/files/spampd/run]: Please change `exec /usr/bin/...` to `exec spampd ...`

File: srcpkgs/spampd/template [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	pod2man --section=8 --center="Spam Proxy Daemon" spampd.pl > spampd.8
+}
+
+do_install() {
+	vbin spampd.pl spampd
+	vman spampd.8
+}
+
+post_install() {
+	vsv spampd
```
Comment (xtraeme) [inline srcpkgs/spampd/template]: There's no reason to use `post_install` just to install the service, move this to `do_install`.

**Comment by Gottox** [comment]
ToDo: 
* [ ] fix crossbuild of perl-NetAddr-IP

**Comment by Vaelatern** [comment]
Bump @Gottox, will crossbuild be fixed or will we fix it after it's in the tree?

**Pull request closed without merging**